### PR TITLE
buy now accordion first version

### PR DESCRIPTION
**Pull request by MoiOcanas**
Updates on last commit for this PR:

- New styles to the buy now form (including the card appearance and changes when it opens and closes).

- Changes to the bulk order form (connection to netlify and new information added to dropdowns).

- The custom card has been hidden for now.

- Metafield "buy_now", is used to define which form will be open in each product (use of ShopifyAdmin query and new props).

- Changes of styles in modules.

- New utils.

- New components.

### CODE REVIEW

**Review by eduardorr97 — APPROVED** [review]

**Comment by eduardorr97** [comment]
@MoiOcanas some comments on this PR
- we need to make sure that the bulk order form name is dynamic so that netlify will create a unique form for each product
- use the values in the form that appears [here](https://www.confirmbiosciences.com/products/saliva-drug-tests/salivaconfirm-saliva-drug-test-premium-no-thc/) to fill in the `state` and `how often?` dropdowns
- the buy now accordion when collapsed needs to look like [this](https://share.getcloudapp.com/5zuwdpQ7)

**Comment by eduardorr97** [comment]
screenshot_1


Hide that whole `Custom Tests Configuration` box for now, please

File: web/src/components/products/ProductActions/components/BulkOrderingForm/BulkOrderingForm.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                    required
+                    name="email"
+                    className={s.formControl}
+                    placeholder="Email"
+                  />
+                </Col>
+                <Col xs={12} sm={6}>
+                  <Form.Control as="select" className={s.formControl} required>
+                    <option selected disabled>Country</option>
+                  </Form.Control>
```
Comment (eduardorr97) [inline web/src/components/products/ProductActions/components/BulkOrderingForm/BulkOrderingForm.tsx:87]: We need to show actual countries here

Comment (MoiOcanas) [reply web/src/components/products/ProductActions/components/BulkOrderingForm/BulkOrderingForm.tsx]: done, I also added a conditional using a hook that only shows the states when the selected country is united states

**Pull request merged**